plugins/code-versions/src/utils/diff/inline-diff.ts

```diff
@@ -3,13 +3,83 @@ import type { InlineDiff } from "./types"
 
```

Comment:
The changes here come from adding the "marks" on changes, rather than highlighting the text. That resulted in having to much highlighted before (leading spaces for example)